modopt/opt/proximity.py

```diff
@@ -1042,3 +1042,69 @@ def _cost_method(self, *args, **kwargs):
             print(' - K-SUPPORT NORM (X):', cost_val)
 
         return cost_val
+
+
+class GroupLASSO(ProximityParent):
```

Comment:
Is there a reference we can add for this?

Reply:
Hmm yes, although ill have to search for the first paper

Reply:
It's not essential, but if we have the information it's always better.